_src/pl/static/js/checkbox-replace.js

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Agreed. @AlasdairSwan has been helpful so far, so perhaps he'll give it a whirl?